lib/logstash/outputs/opensearch.rb

```diff
@@ -200,6 +200,13 @@ class LogStash::Outputs::OpenSearch < LogStash::Outputs::Base
   # here like `pipeline => "%{INGEST_PIPELINE}"`
   config :pipeline, :validate => :string, :default => nil
 
+  # If enabled, skip the GET request to the Healthcheck endpoint to get the server version.
+  # The HEAD request is still done to check if the server is alive. 
```

Comment:
What is `HEAD` request referring to here?

Reply:
The healthcheck! function in pool.rb makes 2 request, one via healhealth_check_request() which sends a HEAD to check connectivity. Then a GET via get_version(url) which receives the info json and parses out the version.

Reply:
Should we rename this property? There is somewhat of a health-check with the `HEAD` request. It isn't quite the same as checking the cluster health, but is somewhat similar. Maybe it should be `skip_version_check`?

Or an alternative would be to skip it only if `default_server_major_version` is not defined.